recipes/geotrans/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+from conans import ConanFile, CMake, tools
+import os
+import shutil
+
+
+class GeotransConan(ConanFile):
+    name = "geotrans"
+    version = "3.8"
+    license = (
+        "NGA GEOTRANS ToS (https://earth-info.nga.mil/php/download.php?file=wgs-terms)"
+    )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://earth-info.nga.mil/"
+    description = "MSP GEOTRANS is the NGA and DOD approved coordinate converter and datum translator."
+    topics = (
+        "geotrans",
+        "geodesic",
+        "geographic",
+        "coordinate",
+        "datum",
+        "geodetic",
+        "conversion",
+        "transformation",
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
```

Comment:
```suggestion
    def configure(self):
        if self.options.shared:
            del self.options.fPIC
         # And if this is a C library:
        del self.settings.compiler.libcxx
        del self.settings.compiler.cppstd
```

This will fix your current hook check.